test/utils.ts

```diff
@@ -41,6 +44,22 @@ export const waitForSpy = (spy: SinonSpy, property = 'called') => {
   });
 };
 
+/**
+ * Creates and returns an os-specific directory for temp files.
+ */
+export const getTempDir = (unique: boolean) => {
```

Comment:
Is there a case where we don't want the temporary directory to **NOT** be unique?